src/lib/models/reflections/abstract.ts

```diff
@@ -216,6 +216,10 @@ export class ReflectionFlags extends Array<string> {
         return this.hasFlag(ReflectionFlag.Readonly);
     }
 
+    get isExported() {
+        return TODO as boolean;
+    }
```

Comment:
This was being called from templates but not declared anywhere.  Do you know what the implementation should be?

Reply:
This was removed in 0.20, it's always true now.